recipes/tip/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "tip" %}
+{% set version = "v1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4ba6f6a9e652fa48a99534997aaaa0921a94af0ef2d65ce909b6825c06a2b259
+
+build:
+  skip: true  # [win and vc<14 or osx]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('tip', max_pin='x.x') }}
+  script_env: 
+    - TIP_VERSION={{ version }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make ==4.3  # [unix]
+    - ninja ==1.10.2
+    - cmake ==3.21.3
+    - pkgconfig ==1.5.5 
+    - libpcap ==1.10.1
+    - arrow-cpp ==6.0.2
+    - spdlog ==1.8.5
+    - yaml-cpp ==0.6.3
+    - gmock ==1.10.0
+    - gtest ==1.10.0
+    - libtins ==4.2
+
+  host:
+    - libpcap ==1.10.1
+    - arrow-cpp ==6.0.2
+    - spdlog ==1.8.5
+    - yaml-cpp ==0.6.3
+    - gmock ==1.10.0
+    - gtest ==1.10.0
+    - libtins ==4.2
+
+  run:
+    - libpcap ==1.10.1
+    - arrow-cpp ==6.0.2
+    - spdlog ==1.8.5
+    - yaml-cpp ==0.6.3
+    - gmock ==1.10.0
+    - gtest ==1.10.0
+    - libtins ==4.2
+
+test:
+  commands:
+    - tests
```

Comment:
This section is actually mostly for packaging tests, so the focus should be on existence tests for libraries and headers along with testing that your user facing CLIs are working. i.e. run the all of the `tip_parse -h` and similar commands here.

Do you need your end user to be able to run this `tests` command? You can run any unit tests that you feel is necessary to validate the build from the build script. However, running the full unittest suite shouldn't be necessary because it is the responsibility of upstream to be running a full tests. Here we just want to check that we got all the compiler settings were compatible, etc.

Reply:
I will push a commit which executes tests in the build scripts and update for packaging related as described. 

Per a previous comment, we do want to keep `tests` around for users to run. We do a lot of juggling of conda channels in containers, followed by env solving then creation at deploy target. Having the tests is an easy way to gain confidence that one of many components is functional.

Reply:
If you want to keep the tests for users to run, then there is no need to execute during the build phase. Keep doing it during the testing phase.

Reply:
I'm getting test failures during execution of lines such as `tip_parse -h` because the CLI library which is ultimately responsible for printing to stdout cannot get the terminal size. This can only be fixed in the source so I am omitting such tests for now.